l10n_it_account_stamp/models/account_move.py

```diff
@@ -71,13 +70,16 @@ def add_tax_stamp_line(self):
                     _("Missing account income configuration for %s")
                     % stamp_product_id.name
                 )
+            price_unit = inv.company_currency_id._convert(
+                stamp_product_id.list_price, inv.currency_id, inv.company_id, inv.date
+            )
```

Comment:
Credo che il campo `list_price` del prodotto sia nella valuta del prodotto quindi andrebbe convertito dalla valuta del prodotto alla valuta della fattura, analogamente a quanto fatto in https://github.com/odoo/odoo/blob/03d2e58f0d9e3beaf6ff8c4486675b4613b88283/addons/sale/models/product_template.py#L266-L269